### ISSUE TITLE
Pass pseudo parameters as CLI arguments

### ISSUE THREAD

**Issue by sjparkinson** [martysweet/cfn-lint#25]
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/pseudo-parameter-reference.html

To avoid:

```
Resource: Resources > ConsulLoadBalancerSecurityGroup > Properties > VpcId
Message: Could not find value in map 123456789012|eu-west-1|VpcId. Have you tried specifying input parameters?
Documentation: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-base64.html
```

```yaml
'000000000000':
  eu-west-1:
    VpcId: vpc-00000000
```

**Comment by martysweet** [comment]
Hi @sjparkinson,

This seems like a sensible suggestion. Looks like the following work would have to be done.

- [ ] Split out cfn-lint/data/aws_ref_override.json into separate files, this seems to mix two different purposes at the moment, or at the least clarify the different data types within the file. It's not clear if the Refs are returning the string of `arn` or another module is expecting an `arn` type.
- [ ] Add another flag to the CLI interface to overwrite the default values specified in a .json file.

Should have a chance to implement this later in the week.

**Comment by martysweet** [comment]
Hi @sjparkinson,

This feature is now implemented in v1.1.1. You can use it like the following:

`cfn-lint validate my_template.yaml --pseudo AWS::StackName="My-Stack"`

`cfn-lint validate my_template.yaml --parameters key="my value" --pseudo AWS::Region=ap-northeast-1,AWS::AccountId=000000000000`
